conf/logs.json

```diff
@@ -212,5 +214,89 @@
         "owner": "integer"
       }
     }
+  },
+  "carbonblack:event01": {
```

Comment:
also this was a constructed log since didn't know if we were concerned about publishing CB formats. I now know that's not the case, so will be updating these will valid CB schemas